Header header

float64 computation_time
float64 freespace_time
float64 cost
float64 kkt
int32 iterations

float64 wC
float64 wL
float64 wV
float64 wW

float64 window
int32 search_points
int32 enable_output

float64 vRef

float64 obstacle_distance1
float64 obstacle_distance2

#obstacle_feed/Obstacles obstacles

float64 obstx_0
float64 obsty_0
float64 obsth_0
float64 obsta_0
float64 obstb_0

float64 obstx_1
float64 obsty_1
float64 obsth_1
float64 obsta_1
float64 obstb_1

std_msgs/Float64MultiArray contour_errors

geometry_msgs/Twist computed_control
nav_msgs/Path reference_path
nav_msgs/Path prediction_horizon
